CHANGES.md

```diff
@@ -10,6 +10,13 @@
       now return a `Parser` (from `XMonad.Util.Parser`) instead of a
       `ReadP`.
 
+  * `XMonad.Config.{Arossato,Bluetile,Dmwit,Droundy,Monad,Prime,Saegesser,Sjanssen}`
+
+    - Deprecated all of these modules.  The user-specific configuration
+      modules will continue to live on [on the website].
```

Comment:
double "on"

Reply:
I'd say that is actually correct: "live on" and "on the website" are separate clauses. Might find a way to rephrase though.

Reply:
Oh indeed, nevermind then. Sorry :-)

Reply:
Yeah this is perhaps a bit awkwardly phrased...  0e28874eb60715903d27a794ad36dbab00547253 rephrases it a tad